printer_mods/KevinAkaSam/VEFACH/README.md

```diff
@@ -0,0 +1,40 @@
+<img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/VEFACH.png" width="300">
+
+# VEFACH - Voron Exhaust Filter Activated Coal + Hepa
+
+<img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD1.png" width="400"/> <img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD2.png" width="400"/>
+
+<img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD3.png" width="400"/> <img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD4.png" width="400"/>
+
+### Why?
+Since the printer produces many small particles, the goal is to filter out as many of them as possible. A few months ago my friend Deutherius had the idea to equip the original exhaust fan with activated coal and a Hepa filter, so I designed this filter inserts. The activated carbon removes e.g. the smell of ABS and the Hepa filter catches most of the small particles.
```

Comment:
Please replace `I designed this filter inserts` with `I designed these filter inserts`